k6_node/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+version: "2.4"
+services:
+  k6_node:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    domainname: ${LOCAL_DOMAIN}
+    ports:
+      - 2222:2222
+    hostname: k6-node
+    container_name: k6-node
+    restart: on-failure
+    networks:
+      k6_node_int:
+      internet:
+        ipv4_address: ${IPV4_PREFIX}.102
+    stop_signal: SIGKILL
+    env_file: [ ".env" ]
+    volumes:
```

Comment:
I think it's a good idea to add here hosts file:
`- ./../../vendor/hosts:/etc/hosts`

Reply:
Added